application/routes/vnc.py

```diff
@@ -97,15 +98,19 @@ def generate():
 
 @api.route('/vncpasswd', methods=['POST'])
 def change_vncpasswd():
+    logger.debug("in route /vncpasswd")
```

Comment:
I'm kinda sure there's a way to enable route logging in Flask so we don't have to manually add logs like this. We can leave the enablement To another PR.
```suggestion
```